portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +25,32 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  //data structure to hold all comments:
+  private Vector allComments = new Vector();
```

Comment:
nit: ArrayList is preferred over Vector.

Reply:
Fixed, thanks